### ISSUE TITLE
mAP not increasing

### ISSUE THREAD

**Issue by kai339** [dmlc/gluon-cv#1393]
Hi, I'm using yolo for object-detection on my custom dataset. Every image contains one or more than one object (same object).
I have ~600 training images and ~130 validation images. I trained the model for ~110 epochs. The mAP I got seems weird, instead of increase gradually, it keeps fluctuating. Here is the best map I've got:
(I set the learning rate to 0.0001, but the default learning rate 0.001 looks similarly)
**0000:	0.0028
0002:	0.0379
0003:	0.1121
0004:	0.1422
0023:	0.1480
0028:	0.1875
0031:	0.1941
0035:	0.2334
0069:	0.2464
0082:	0.2471
0091:	0.1517
0094:	0.1867
0099:	0.2038**

Should I train for more epoch? But it seems to me that the mAP isn't going to improve anymore.
Any suggestions on how to improve this?
**Thanks for any help.**

**Comment by Aktcob** [comment]
suggest that u test the map of training datasets firstly.

**Comment by KaiWang66** [comment]
@Aktcob, **Thank you for help**! I test the training images(~600 images) and the map I got is 0.8979420897945736. Is it possible that something wrong with validation images? But I divide all images I have into training and testing images and divide training images into training set and validation set. Since all images contain the same object, validation images shouldn't differ significantly.
